### ISSUE TITLE
ioctl node delegate does not show name on testnet, but works fine on mainnet

### ISSUE THREAD

**Issue by dustinxie** [iotexproject/iotex-core#3880]
### What version of iotex-core image (or code branch) are you using?
latest

### What operating system and processor architecture are you using?
macOS

### What did you do? If possible, provide a recipe for reproducing the error.
```
ioctl node delegate -a --endpoint=api.testnet.iotex.one:443
```

### What did you expect to see?
list of delegates with names displayed

### What did you see instead?
no names are displayed, while it works fine for mainnet
```
ioctl node delegate -a --endpoint=api.iotex.one:443
```

**Comment by millken** [comment]
using a local snapshot db test, name can be shown. may the db on the line was broken
![sn9I6kVacU](https://github.com/iotexproject/iotex-core/assets/176581/02d5e13b-f53b-42db-bf3a-832ba89546eb)

**Comment by millken** [comment]
fixed in #3882
